ONSdigital/takeon-business-layer
# Modified error message of Date Adjustment functionality

# Description
Modified DateAdjustment Error message where it says 'Selective Editing' in one of the Java file

# How to test?
Execute Date Adjustment without a configuration in a database. You can see correct error message after the above changes
